src/common.js

```diff
@@ -428,22 +422,36 @@ exports.readFromPipe = _readFromPipe;
 var DEFAULT_WRAP_OPTIONS = {
   allowGlobbing: true,
   canReceivePipe: false,
-  cmdOptions: false,
+  cmdOptions: null,
   globStart: 1,
   pipeOnly: false,
-  unix: true,
   wrapOutput: true,
-  overWrite: false,
+  unix: true,
```

Comment:
Looking at the source, `unix` is only used by `exec`, because `exec` args differ from the other commands. I think now would be a good chance to improve this.

It makes sense to expose the ability to skip manipulating the args, so let's rename the option to make that clearer. Something like `preserveArgs`?

Reply:
I was planning to hide the `unix` option in a later PR. Renaming is ok too, but I don't see a need to expose it (we already offer ways to disable globbing and option parsing, which are the most expensive argument manipulations).

Reply:
Ok, that's fine.